prow/jobs/kyma/kyma-kind-integration.yaml

```diff
@@ -197,7 +197,7 @@ periodics:
       - --tune-inotify
       - --start-docker
       - --kubernetes-version
-      - v1.16.3
+      - v1.16.4
```

Comment:
I'm not sure if that version is available at https://hub.docker.com/r/kindest/node/.

Reply:
Ok, I see `v1.16.4` was released 4h ago

Reply:
Reverted